blog/blog.html

```diff
@@ -1,52 +1,22 @@
 <!DOCTYPE html>
-
-<html class="no-js" lang="en">
```

Comment:
I dont know what "no-js" is for but it's not used or mentioned anywhere in your project. Let's keep it tidy and clean up any unnecessary code